gradio/blocks.py

```diff
@@ -1727,10 +1728,13 @@ def launch(
         show_api: bool = True,
         allowed_paths: list[str] | None = None,
         blocked_paths: list[str] | None = None,
+        *,
```

Comment:
what about making all of the arguments keyword-only in launch()? I would support doing this throughout the codebase for 4.0

Reply:
I didn't want to be too aggressive here since it'll just cause unnecessary breaking of parameters. But let me move it a few positions back